crates/wasmtime/src/ref.rs

```diff
@@ -36,6 +36,11 @@ impl ExternRef {
         &*self.inner
```

Comment:
Looks like this file become executable?

Reply:
I really need to figure out why the heck this is happening...